CMakeLists.txt

```diff
@@ -31,6 +31,9 @@ target_link_libraries(dump_xsettings PRIVATE libxsettingsd X11::X11)
 install(TARGETS xsettingsd dump_xsettings DESTINATION ${CMAKE_INSTALL_BINDIR})
 install(FILES xsettingsd.1 dump_xsettings.1 DESTINATION ${CMAKE_INSTALL_MANDIR}/man1)
 
+configure_file(xsettingsd.service.in xsettingsd.service)
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/xsettingsd.service DESTINATION lib/systemd/user)
```

Comment:
should `${CMAKE_CURRENT_BINARY_DIR}/xsettingsd.service` be double-quoted to handle spaces being present in `CMAKE_CURRENT_BINARY_DIR`? again, i don't know anything about cmake and how similar this file is to normal sh, but from https://cmake.org/pipermail/cmake/2017-September/066338.html, it sounds like might be recommended.

Reply:
You are right. It's just that spaces in paths is only common on Windows, and since xsettingsd doesn't run there, I omitted the quotes. Fixed.